SPECS/dbus/dbus.spec

```diff
@@ -20,6 +20,8 @@ Requires:       xz
 # In real-life situations systemd will always be present and thus installed.
 Recommends:     systemd
 Provides:       dbus-libs = %{version}-%{release}
+# De-entagle SPECS-EXTENDED dependency
```

Comment:
I think this comment is not going to say much to anyone looking at it later on without the larger context. Also, the main reason this is here, is that we haven't enabled building with X11, yet, and want to keep some extended package builds unblocked. I'd suggest saying that here instead.